backend-opal/src/database/migrations/1574590998449_create_table_users.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
@martinupset let's call this migration create table watchlists

Reply:
ok